README.md

```diff
@@ -1,52 +1,49 @@
-# clidoscope-demo
+# Develop Across Salesforce With One CLI
 
-An experiment to explore a new taxonomy for all Salesforce clouds.
+Welcome to the Salesforce CLI Unification Demo CLI repo. We’re designing a new taxonomy and structure for Salesforce CLI with the intention of unifying all of Salesforce’s products under one CLI. Our goal is to provide a seamless developer experience when building applications on any Salesforce product as well as across products. The demo CLI is an interactive simulation setup with the new taxonomy and common workflows, it doesn't actually do anything or change your environment. Only a handful of commands have full help content. Please test it out and tell us what you think.
+
+## Strategy
+
+See Strategy and Structure for more details 
 
 ## Install
 
 ```bash
 npm install --global @salesforce/sf-demo
 sf help
```

Comment:
This doesn't work right now (although maybe it will later when this demo cli fully implements the recent help changes?).  This works today:

sf --help